radio/src/gui/480x272/menus.cpp

```diff
@@ -66,6 +66,32 @@ void pushMenu(MenuHandlerFunc newMenu)
   TRACE("pushMenu(%d, %p)", menuLevel, newMenu);
 }
 
+void readModelNotes() {
+
+  LED_ERROR_BEGIN();
+
+  strcpy(s_text_file, MODELS_PATH "/");
+  char *buf = strcat_currentmodelname(&s_text_file[sizeof(MODELS_PATH)]);
+  strcpy(buf, TEXT_EXT);
+  if (!isFileAvailable(s_text_file)) {
+    char *buf = strAppendFilename(&s_text_file[sizeof(MODELS_PATH)], g_eeGeneral.currModelFilename, LEN_MODEL_FILENAME);
+    strcpy(buf, TEXT_EXT);
+  }
+
+  clearKeyEvents();
+  event_t event = EVT_ENTRY;
+  while (event != EVT_KEY_FIRST(KEY_EXIT)) {
+    lcdRefreshWait();
+    lcdClear();
+    menuTextView(event);
+    event = getEvent();
+    lcdRefresh();
+    wdt_reset();
```

Comment:
did someone test it? there is no delay between 2 LCD refresh, I hope it's OK

Reply:
I borrowed a x12 to test it, it seems to work fine, maybe a x10 test too would be nice.

Reply:
ok I will do it